src/test/resources/features/Registration.feature

```diff
@@ -59,4 +59,41 @@ Feature: Register for ECL
   Scenario: User does not select their entity type
     Given I am signed in to the registration journey
     When I do not select an option for my entity type
-    Then I should see an error that says Please select your entity type
\ No newline at end of file
+    Then I should see an error that says Please select your entity type
+
+  Scenario: User does not select their start AML regulated activity questions
+    Given I am signed in to the registration journey
+    When I do not select an option for my AML regulated activity questions
+    Then I should see an error that says Select an answer
+
+  Scenario: User select No for their start AML regulated activity questions
+    Given I am signed in to the registration journey
+    When I select No for my AML regulated activity questions
+    Then I should be on the page that says What is your business sector?
```

Comment:
Should this not be part of the happy part full registration journey scenario?

Reply:
done